esp/services/ws_workunits/ws_workunitsService.cpp

```diff
@@ -5014,3 +5014,442 @@ bool CWsWorkunitsEx::onWUGetArchiveFile(IEspContext &context, IEspWUGetArchiveFi
     }
     return true;
 }
+
+//Copy from eclcmd_common.cpp
+bool CWsWorkunitsEx::isValidPriorityValue(const char *value)
```

Comment:
It would be nice if these were shared between here and eclcmd_common.cpp somewhere.